### PR TITLE
ET -6: Metaphor styling on register page.

### PR DESCRIPTION
**Pull request by frlzjosh**
-The register page will look weird but I will fix it .
-Logout still doesn't work
-Gabe Alex Luis and other helped with fixing a lot of things in the dev and with stuff that'll help us in the future

### CODE REVIEW

File: resources/views/auth/register.blade.php [hunk 1]
```diff
@@ -67,10 +72,10 @@
                         </div>
 
                         <div class="form-group{{ $errors->has('password') ? ' has-error' : '' }}">
-                            <label for="password" class="col-md-4 control-label">Password</label>
+                            <h3 for="password" >Password</h3>
 
                             <div class="col-md-6">
-                                <input id="password" type="password" class="form-control" name="password" required>
+                                <input id="password" type="password" class="form-control" name="password" required placeholder="Password ...">
```
Comment (RahilHashim) [inline resources/views/auth/register.blade.php]: make the lower case password, Password

File: resources/views/auth/register.blade.php [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+                    <form role="form" method="POST" action="{{ route('register') }}">
                         {{ csrf_field() }}
 
                         <div class="form-group{{ $errors->has('first_name') ? ' has-error' : '' }}">
-                            <label for="first_name" class="col-md-4 control-label">First Name</label>
+                            <h4 for="first_name">First Name</h4>
 
                             <div class="col-md-6">
-                                <input id="first_name" type="text" class="form-control" name="first_name" value="{{ old('first_name') }}" required autofocus>
+                                <input id="first_name" type="text" class="form-control" name="first_name" value="{{ old('first_name') }}" required autofocus placeholder="First Name here..">
```
Comment (RahilHashim) [inline resources/views/auth/register.blade.php]: dont need here

File: resources/views/auth/register.blade.php [hunk 3]
```diff
@@ -25,10 +30,10 @@
                         </div>
 
                         <div class="form-group{{ $errors->has('last_name') ? ' has-error' : '' }}">
-                            <label for="last_name" class="col-md-4 control-label">Last Name</label>
+                            <h4 for="last_name">Last Name</h4>
 
                             <div class="col-md-6">
-                                <input id="last_name" type="text" class="form-control" name="last_name" value="{{ old('last_name') }}" required autofocus>
+                                <input id="last_name" type="text" class="form-control" name="last_name" value="{{ old('last_name') }}" required autofocus placeholder="Last Name here ...">
```
Comment (RahilHashim) [inline resources/views/auth/register.blade.php]: same thing dont put here

File: resources/views/auth/register.blade.php [hunk 4]
```diff
@@ -39,11 +44,11 @@
                         </div>
 
                         <div class="form-group{{ $errors->has('user_name') ? ' has-error' : '' }}">
-                            <label for="user_name" class="col-md-4 control-label">User Name</label>
+                            <h3 for="user_name">User Name</h3>
 
                             <div class="col-md-6">
-                                <input id="user_name" type="text" class="form-control" name="user_name" value="{{ old('user_name') }}" required autofocus>
-
+                                <input id="user_name" type="text" class="form-control" name="user_name" value="{{ old('user_name') }}" required autofocus placeholder="Your Username ...">
```
Comment (RahilHashim) [inline resources/views/auth/register.blade.php]: Username . . .

Comment (RahilHashim) [inline resources/views/auth/register.blade.php]: no . . .

File: resources/views/auth/register.blade.php [hunk 5]
```diff
@@ -1,20 +1,25 @@
 @extends('layouts.master')
```
Comment (RahilHashim) [inline resources/views/auth/register.blade.php]: Fix the Overall look of the page / style

File: resources/views/welcome.blade.php [hunk 6]
```diff
@@ -1,105 +1,106 @@
-
 <!doctype html>
 @extends ('layouts.master')
 
 @section('title')
     Laravel Project
+    {{ $hello }}
```
Comment (RahilHashim) [inline resources/views/welcome.blade.php]: take out

File: resources/views/welcome.blade.php [hunk 7]
```diff
@@ -1,105 +1,106 @@
-
 <!doctype html>
```
Comment (RahilHashim) [inline resources/views/welcome.blade.php]: fix this page . . .

File: routes/web.php [hunk 8]
```diff
@@ -15,6 +15,10 @@
     return view('welcome');
 });
 
+//Route::get('luis', function() {
```
Comment (RahilHashim) [inline routes/web.php]: comment this out / delete?

File: resources/views/auth/register.blade.php (hunk 5 above)
Comment (frlzjosh) [inline resources/views/auth/register.blade.php]: I did it

**Pull request closed without merging**